tests/testthat/test-add_global_p.tbl_regression.R

```diff
@@ -0,0 +1,123 @@
+test_that("add_global_p.tbl_regression works with standard use", {
```

Comment:
Can you add a skip if the following pkgs are not installed? Can you also add this to the top of the `add_global_p.tbl_uvregression()` testing function?
broom.helpers
cardx
car
aod

Reply:
nevermind I just added them and i'll merge the PR! thank you!